packages/documentation-site/patternfly-docs/content/layouts/compass/dashboard.tsx

```diff
@@ -0,0 +1,278 @@
+          </p>
+          <ActionList>
+            <ActionListGroup>
+              <ActionListItem>
+                <Button variant="primary">Upgrade today</Button>
+              </ActionListItem>
+              <ActionListItem>
+                <Button variant="secondary">Talk to a Red Hatter</Button>
+              </ActionListItem>
+            </ActionListGroup>
+          </ActionList>
+        </Content>
+      </CompassHero>
+      <CompassContent id="main">
+        <Grid hasGutter style={{ maxHeight: 'none' }}>
+          <GridItem span={12} sm={12} md={6} lg={4} xl={3} rowSpan={4}>
+            <ClusterDetailsCard />
+          </GridItem>
+          <GridItem span={12} sm={12} md={6} lg={4} xl={3} rowSpan={2}>
+            <AnimationsOverviewClusterInventory />
+          </GridItem>
+          <GridItem span={12} sm={12} md={6} lg={4} xl={3} rowSpan={2}>
+            <AnimationsOverviewStorage />
+          </GridItem>
+          <GridItem span={12} sm={12} md={12} lg={8} xl={3} rowSpan={2}>
+            <AnimationsOverviewMemoryUtilization />
+          </GridItem>
+          <GridItem span={12} sm={12} md={12} lg={12} xl={3} rowSpan={2}>
+            <AnimationsOverviewNetworkActivity />
+          </GridItem>
+          <GridItem span={12} sm={12} md={12} lg={8} xl={6} rowSpan={2}>
+            <RecentActivityCard />
+          </GridItem>
+        </Grid>
+      </CompassContent>
+    </>
+  );
+
+  const eastContent = (
+    <CompassPanel isPill>
+      <ActionList isIconList isVertical>
+        <ActionListGroup>
+          <ActionListItem>
+            <Tooltip content="Help 2">
+              <Button variant="plain" icon={<OutlinedQuestionCircleIcon />} aria-label="Help 2" isCircle />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Add">
+              <Button variant="plain" icon={<OutlinedPlusSquare />} aria-label="Add" isCircle />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Help 3">
+              <Button variant="plain" icon={<OutlinedQuestionCircleIcon />} aria-label="Help 3" isCircle />
+            </Tooltip>
+          </ActionListItem>
+        </ActionListGroup>
+      </ActionList>
+    </CompassPanel>
+  );
+
+  const handleSendMessage = () => {
+    setIsThinking(true);
+    setTimeout(() => {
+      setIsThinking(false);
+    }, 10000); // 10 seconds
+  };
+
+  const southContent = (
+    <CompassMessageBar>
+      <CompassPanel isPill hasNoPadding hasNoBorder>
+        <MessageBar
+          isCompact
+          onSendMessage={handleSendMessage}
+          alwayShowSendButton
+          hasAttachButton={false}
+          hasAiIndicator
+          isThinking={isThinking}
+        />
+      </CompassPanel>
+      <div className="pf-v6-screen-reader" aria-live="assertive">{isThinking && "AI is thinking..."}</div>
```

Comment:
```suggestion
      <div className="pf-v6-screen-reader" aria-live="polite">{isThinking && "AI is thinking..."}</div>
```

Just to avoid using assertive if wew don't need to; I'm not sure if there'd be a case where this would need to override any other announcement happening on the page, and since it isn't time sensitive or anything we can probbaly make do with this announcement occurring after any other ones.